fuze-youth-conference.html

```diff
@@ -11,7 +11,7 @@ <h1>fuZe Youth Allyship Conference</h1>
     <p><strong>Join area high school students on Sunday, February 28th from 4:00-5:30pm for <u>fuZe: An Interfaith and Community Youth Allyship Conference</u>.</strong></p>
     <p>We will foster <strong>understanding and empathy</strong> amongst a diverse Gen Z group. Participants will gain an understanding of how to challenge their own perspectives, the importance of community service, and techniques to <strong>“become comfortable with being uncomfortable”</strong> with the inclusion of new faces in their own social circles. <strong>Together</strong>, we will learn that even with different faiths and cultures, we can be brought together with <strong>shared humanity</strong>. This conference will lay the foundation for relationships based on trust, consistency, and accountability for embracing each other’s differences, creating a <strong>future generation of allies</strong> and a path to <strong>social justice</strong>.</p>
     <p><strong>We will meet on February 28 online on Zoom, but we look forward to future events bringing conference participants together in-person, when possible.</strong></p>
-    <p><strong>Interested? <a href="mailto:fuzeyouthconference@gmail.com">Email</a> with questions and to register.</strong></p>
+    <p><mark><strong>Interested? <a href="mailto:fuzeyouthconference@gmail.com">Email</a> with questions or register <a href="https://bit.ly/fuZeyouthconference">here</a>.</strong></mark></p>
```

Comment:
I would prefer to do highlighting with CSS instead. No big deal here though.

Reply:
Should I make a new class called `highlight` instead? What is the best way to learn HTML and CSS best practices?

Reply:
That sounds good to me.
Typically it's just good to keep the languages to their goals—content is HTML, appearance is CSS, functionality JS. Before CSS was invented/standardized people created a bunch of HTML tags to represent things that should now go through CSS, like `<center>` or `<marquee>`. `<mark>` is kind of a middle ground since it's not that different from things like `<strong>` and `<em>` which are still fine to use, but because there's a lot of ways to display highlighting I think we should do it manually through CSS>

Reply:
Thank you for the explanation. I did not realize that there were HTML elements that should not be used.